xm-commons-domainevent/src/main/java/com/icthh/xm/commons/domainevent/service/filter/DomainEventProviderFactory.java

```diff
@@ -12,7 +12,6 @@
 
 @Component
 @RequiredArgsConstructor
-@ConditionalOnProperty(value = "application.domain-event.enabled", havingValue = "true")
 public class DomainEventProviderFactory {
```

Comment:
why?

Reply:
not relevant, since the entire configuration starts with DomainEventConfig in which the given @ConditionalOnProperty is specified

Reply:
this bean will be started by ComponentScan

Reply:
fixed